### ISSUE TITLE
Fine Tuning GR00T

### ISSUE THREAD

**Issue by kjyothiswaroop** [NVlabs/GR00T-WholeBodyControl#167]
**Description**

I am currently fine tuning GR00T with the instructions given in the docs i.e. : 
```
**export NUM_GPUS=1**

uv run python \
    gr00t/experiment/launch_finetune.py \
    --base-model-path nvidia/GR00T-N1.7-3B \
    --dataset-path /path/to/your/collected_dataset \
    --embodiment-tag **UNITREE_G1_SONIC_DEX1** \
    --modality-config-path gr00t/configs/data/embodiment_configs.py \
    --num-gpus $NUM_GPUS \
    --output-dir /path/to/output \
    --save-total-limit 5 \
    --save-steps 5000 \
    --max-steps 20000 \
    --use-wandb \
    --global-batch-size 32 \
    --color-jitter-params brightness 0.3 contrast 0.4 saturation 0.5 hue 0.08 \
    --dataloader-num-workers 4
```

The only difference is that I have changed the embodiment config, instead of using the UNITREE_G1_SONIC, i am using a custom embodiment i made called UNITREE_G1_SONIC_DEX1. (Added the Dex_1_1 gripper and a 2DOF head for the G1)

I collected about 50 episodes for a simple pick and place task and used a **single GPU** to finetune GR00T.

**My concern right now, is that the model finetunes quite quickly when compare to other learning methods like Pi0.5, ACT etc. I  want to know if this is expected or there me something wrong i am doing, because the docs say Multiple GPUs(4+) required. With 50 episodes it took only about 4 hours to train.**

**Setup**

Using the docker container to train the model.

| | |
|---|---|
| **Model** | GR00T-N1.7-3B (`nvidia/GR00T-N1.7-3B`) |
| **isaac-gr00t version** | 0.1.0 |
| **GPU** | 1× NVIDIA RTX 6000 Ada Generation (48 GB VRAM each) |
| **Driver** | 550.127.05 |
| **CUDA** | 12.4 |
| **cuDNN** | 9.4.0 |
| **PyTorch** | 2.4.1 |
| **CPU** | AMD Ryzen Threadripper PRO 5995WX (64-core / 128-thread) |
| **RAM** | 995 GB |
| **OS / Kernel** | Linux 6.8.0-31-generic |
| **Training GPUs used** | 1 (single-GPU fine-tune) |
| **Embodiment tag** | `UNITREE_G1_SONIC_DEX1` |

**Comment by fcastanedaga** [comment]
Hi! For 50 episodes 4 hours might be a reasonable time, what does your loss curve look like? Have you tried deploying the finetuned checkpoint? What's the performance like?

**Comment by kjyothiswaroop** [comment]
Hey, yeah I did deploy and the inference actually worked! I am pretty surprised that Groot trains very quick in comparision to Pi0.5 or other models we could finetune
